libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat

```diff
@@ -70,8 +72,12 @@ PC12 SPI3_MOSI SPI3
 PC11 SPI3_MISO SPI3
 PC10 SPI3_SCK SPI3
 
-PB5 LED OUTPUT HIGH GPIO(57)
-PB4 TIM3_CH1 TIM3 GPIO(56) ALARM
+
+PB5 LED_BLUE OUTPUT LOW GPIO(0)
+define HAL_GPIO_A_LED_PIN 0
+define HAL_GPIO_B_LED_PIN 1 #dummy assignment required to allow AP_NOTIFY to use board led
+
+#PB4 TIM3_CH1 TIM3 GPIO(58) ALARM
```

Comment:
Why this commented out?

Reply:
mistaken remnant of attempts to get led to work...corrected...